website/docs/seo.md

```diff
@@ -0,0 +1,74 @@
+---
+id: seo
+title: Search engine optimization (SEO)
+sidebar_label: SEO
+keywords:
+  - seo
+  - positioning
+---
+
+Docusaurus supports the search engine optimization in a variety of ways.
+
+```mdx-code-block
+import TOCInline from '@theme/TOCInline';
```

Comment:
I'd remove this because it leads to a duplicate TOC on mobile and wouldn't be consistent with other docs we have

![image](https://user-images.githubusercontent.com/749374/140177162-8c0e071c-425b-41a4-b85b-89c79a61cd42.png)